openpype/hosts/houdini/api/lib.py

```diff
@@ -548,7 +548,7 @@ def get_template_from_value(key, value):
     return parm
 
 
-def get_frame_data(node):
+def get_frame_data(self, node, asset_data=None):
```

Comment:
```suggestion
def get_frame_data(node, asset_data=None):
```
Dont think it's the function under the classes.

Reply:
I mainly did that because I wanted the `get_frame_data` to use the log associated with the caller 

what about 
```python
def get_frame_data(node, asset_data=None, log=None):
    if log is None: 
        log = self.log
```

Reply:
Yup, that seems more reasonable.